nautobot/ipam/forms.py

```diff
@@ -603,11 +603,11 @@ class IPAddressFilterForm(NautobotFilterForm, TenancyFilterForm, StatusModelFilt
         label="IP version",
         widget=StaticSelect2(),
     )
-    mask_length = forms.ChoiceField(
+    mask_length = forms.MultipleChoiceField(
```

Comment:
Change the field and widget attributes to account for `mask_length` filter becoming a `MultiValueNumberFilter`.